features/integration/algod.feature

```diff
@@ -1,48 +1,11 @@
 @algod
 Feature: Algod
   Background:
-    Given an algod client
+    Given an algod v2 client connected to "localhost" port 60000 with token "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
 
   Scenario: Node health
     Then the node should be healthy
 
-  Scenario: Status check
```

Comment:
The [unused steps notebook](https://github.com/algorand/algorand-sdk-testing/blob/master/utils/unused_steps_analysis.ipynb) might aid in validating that all the unused steps have been cleaned up from the SDK's. I can help here either by running the notebook when it's ready to give a spin, and/or elucidating how to use it.

Reply:
Thanks for the heads up - I'll definitely try to give this a spin!